server/schemas/integration_webhook.graphql

```diff
@@ -0,0 +1,63 @@
+type WebhookEvent {
+  model: ID!
+  onCreate: Boolean!
+  onUpdate: Boolean!
+  onDelete: Boolean!
+  onApiAccess: Boolean!
+}
+
+type Webhook implements Node {
+  id: ID!
+  name: String!
+  url: URL!
+  active: Boolean!
+  event: [WebhookEvent!]!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
```

Comment:
```suggestion
type Webhook {
  id: ID!
  name: String!
  url: URL!
  active: Boolean!
  trigger: WebhookTrigger!
  createdAt: DateTime!
  updatedAt: DateTime!
}

type WebhookTrigger {
  onItemCreate: Boolean!
  onItemUpdate: Boolean!
  onItemDelete: Boolean!
  onAssetUpload: Boolean!
  onAssetDeleted: Boolean!
  onItemPublish: Boolean!
  onItemUnpublish: Boolean!
}
```

Webook is an entity, but it is better not to be a root entity. It just belongs to an Integration.